docs/overview/overview.md

```diff
@@ -0,0 +1,96 @@
+# P4 Control Plane
+
+P4 Control Plane (formerly P4-OVS Split Architecture) modularizes P4-OVS
+and reduces coupling between its components, making the code easier to maintain
+and more suitable for upstreaming. It moves the P4-specific components of the
+architecture to a separate process called `infrap4d`.
+
+![P4 Control Plane Architecture](p4-control-plane-architecture.png)
```

Comment:
the diagram shows ovsbd-server can also talk P4 using ovs-p4rt. This is not implemented yet.